x/marker/client/cli/tx.go

```diff
@@ -1053,11 +1053,38 @@ $ %[1]s tx marker account-data hotdogcoin --%[4]s`,
 // GetCmdAddNetAssetValues returns a CLI command for adding/updating marker net asset values.
 func GetCmdAddNetAssetValues() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "add-net-asset-values <denom> " + attrcli.AccountDataFlagsUse,
+		Use:     "add-net-asset-values <denom> <valuation[;valutation...]>",
 		Aliases: []string{"add-navs", "anavs"},
-		Short:   "Add/updates net asset values for a marker",
-		Example: fmt.Sprintf(`$ %[1]s tx marker add-net-asset-values hotdogcoin 1usd,1;2nhash,3`,
-			version.AppName),
+		Short:   "Provide net asset values for a marker",
+		Long: `
+Proved net asset valuations for a marker.  Net asset values are expressed as a ratio
```

Comment:
Might be typo? `Proved net asset valuations` seems like it doesn't read correctly.

Reply:
I completely replaced this section based on feedback from the code rabbit.